### PR TITLE
change slide for Wei 

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random meme for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [x] @jimmieego
- [ ] @intheswim
- [ ] @epitrochoider
- [ ] @githubteacher
- [ ] @skrishn
- [ ] @TheMalaka
- [ ] @weiy0415
- [ ] @shub8703
- [ ] @AshPengelly
- [ ] @mpurohit1
- [ ] @reddyneeta
- [ ] @Lyuye
- [ ] @Lyuye
- [ ] @ak-esri

### PR DESCRIPTION
**Pull request by weiy0415**
Notify others for attention:

Ask questions:
- Q1
- Q2

refer to other issues 
related to #2 
closes #2

### CODE REVIEW

**Review by yuzheyan — APPROVED** [review]
Good

File: _posts/0000-01-16-githubteacher.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Wei's slide!"
 ---
 
 ![fry](https://cloud.githubusercontent.com/assets/16547949/25400891/fb18941a-29c1-11e7-82b9-35e589443a60.jpg)
 {: .center}
 
-Caption here
+Hello World!
```
Comment (yuzheyan) [inline _posts/0000-01-16-githubteacher.md:11]: this should be "-" instead of +

File: _posts/0000-01-16-githubteacher.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Wei's slide!"
```
Comment (yuzheyan) [inline _posts/0000-01-16-githubteacher.md:3]: Is this really your slide?

File: _posts/0000-01-16-githubteacher.md (hunk 1 above)
Comment (weiy0415) [reply _posts/0000-01-16-githubteacher.md:11]: :lollipop:

File: _posts/0000-01-16-githubteacher.md (hunk 2 above)
Comment (weiy0415) [reply _posts/0000-01-16-githubteacher.md:3]: :-1:

File: _posts/0000-01-16-githubteacher.md (hunk 1 above)
Comment (yuzheyan) [reply _posts/0000-01-16-githubteacher.md:11]: 好

Comment (weiy0415) [reply _posts/0000-01-16-githubteacher.md:11]: :new_moon_with_face:

**Review by yuzheyan — APPROVED** [review]

**Pull request closed without merging**